spec/policies/friendships_policy_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe FriendshipPolicy, type: :policy do
+  let(:kurt) { create(:user, username: 'kurt') }
+  let(:mary) { create(:user, username: 'mary') }
+  let(:friendship) { create(:friendship, active_friend: kurt, passive_friend: mary) }
+
+  context 'mary can either confirm or reject the request' do
```

Comment:
Nice context names. However, `mary` says nothing, what is so special about Mary that she can confirm/reject friendship? Is it because she got invitation for friendship? Then use something like : 
 context `user who received friendship request `
  it `can accept request`
  it `can reject request`

Reply:
It applies to all specs:)

Reply:
Done! :) I hope I understood clearly what you meant. I looked into the model specs and requests specs and didn't change any context names as I felt they were clear enough. The context names for these policy specs are yeah rather unclear